### ISSUE TITLE
Notícia #4 360º News #4

### ISSUE THREAD

**Issue by allluz** [w3b3d3v/squad-midiatico-content-PT#55]

**Comment by s4mp41xao** [comment]
# Pai e Filho Sentenciados por Lavagem de Dinheiro Proveniente do Tráfico de Drogas com Bitcoin na Darknet Pela Silk Road

#silkroad #bitcoin #investigation #darknet

Numa reviravolta bizarra entre crime e criptomoedas, uma dupla de pai e filho se encontra nos enredos da justiça. A história envolve lavagem de dinheiro, transações na darknet e uma tentativa de ludibriar as autoridades. Prepare-se enquanto nos aprofundamos na trama por Joseph e Ryan Farace.

---
###### Sumário

1. [As Sentenças: A Queda da Honra Familiar](#as-sentenças-a-queda-da-honra-familiar)
2. [A Saga das Drogas na Darknet: Bitcoin e Xanax Revelados](#a-saga-das-drogas-na-darknet-bitcoin-e-xanax-revelados)
3. [Um Toque de Colaboração: Os Inesquecíveis Bitcoins](#um-toque-de-colaboração-os-inesquecíveis-bitcoins)
4. [Colaboração Pai e Filho: Lavagem atrás das Grades](#colaboração-pai-e-filho-lavagem-atrás-das-grades)
5. [Comunicação Criptográfica: Bitcoins atrás das Grades](#comunicação-criptográfica-bitcoins-atrás-das-grades)
6. [Manobras Offshore: Da Prisão para Contas Estrangeiras](#manobras-offshore-da-prisão-para-contas-estrangeiras)
7. [O Confisco: Milhões em Bitcoin Apreendidos](#o-confisco-milhões-em-bitcoin-apreendidos)
8. [Elogios e Gratidão: Triunfo das Forças de Segurança](#elogios-e-gratidão-triunfo-das-forças-de-segurança)
9. [Olhando para o Futuro: Além da Cortina de Bitcoin](#olhando-para-o-futuro-além-da-cortina-de-bitcoin)


---

### As Sentenças: A Queda da Honra Familiar
O martelo da sentença cai sobre Joseph Farace, um senhor de 72 anos, de Sparks, Maryland, conforme a Juíza do Distrito dos EUA, Deborah K. Chasanow, que determinou 19 meses de prisão federal em [divulgação oficial](https://www.justice.gov/usao-md/pr/father-and-son-sentenced-laundering-drug-trafficking-bitcoin-proceeds-intended-federal) no dia 8 de janeiro. Seu filho, Ryan Farace, um homem de 38 anos com um histórico criminoso anterior, recebe uma sentença mais severa de 54 meses de prisão federal em 5 de janeiro de 2024. O que levou a essa queda familiar?

### A Saga das Drogas na Darknet: Bitcoin e Xanax Revelados
O repertório de Ryan Farace começou em 2018, quando foi condenado por fabricar e distribuir comprimidos de alprazolam, conhecido como "Xanax", através de mercados na darknet. Rastros de criptomoedas revelaram incríveis 9.138 Bitcoins fluindo para carteiras associadas a ele e seu pseudônimo na darknet, "XANAXMAN".

### Um Toque de Colaboração: Os Inesquecíveis Bitcoins
Antes de sua sentença em 2018, Ryan Farace, talvez prevendo o inevitável, colaborou com as autoridades. No entanto, essa cooperação sofreu um revés após a sentença, quando o governo deparou-se com receitas adicionais provenientes do tráfico de drogas em 24 Bitcoins, para ser preciso. Um caso peculiar de perda de memória ou evasão estratégica?

### Colaboração Pai e Filho: Lavagem atrás das Grades
Sem se deixar abalar pelas paredes da prisão, Ryan Farace, com seu pai Joseph Farace, embarcou numa escapada de lavagem de dinheiro de outubro de 2019 a abril de 2021. Enquanto estava detido, Ryan orquestrou a transferência de 71 Bitcoins para exchanges e varejistas online, beneficiando seu pai com cartões-presente no valor de $ 3.341,65.

### Comunicação Criptográfica: Bitcoins atrás das Grades
Comunicando-se por meio de e-mails criptografados e telefones celulares clandestinos, os Farace utilizaram uma linguagem codificada para discutir transferências de Bitcoin. Num toque cinematográfico, Ryan transmitiu um endereço de carteira a seu pai, gravando-o na contracapa de um livro da biblioteca da prisão. Um caso clássico de engenhosidade ou desespero?

### Manobras Offshore: Da Prisão para Contas Estrangeiras
Em agosto de 2020, enquanto cumpria pena, Ryan Farace solicitou a seu pai a transferência de mais de 2.874 Bitcoins para uma conta bancária estrangeira. A troca sigilosa envolveu Ryan enviando o endereço da carteira discretamente escrito num livro da biblioteca da prisão. No entanto, as autoridades estavam um passo à frente.

### O Confisco: Milhões em Bitcoin Apreendidos
Em 10 de fevereiro de 2021, agentes federais confiscaram os 2.874 Bitcoins transferidos por Joseph Farace. O valor de mercado na época variava entre $ 65 milhões e $ 150 milhões. Posteriormente, em 11 de maio de 2021, mais de $ 58 mil Bitcoins, também provenientes do tráfico de drogas de Ryan Farace, que tiveram o mesmo destino.

### Elogios e Gratidão: Triunfo das Forças de Segurança
O Procurador dos Estados Unidos Erek L. Barron e a Procuradora-Geral Adjunta Nicole M. Argentieri estenderam sua gratidão às agências de segurança envolvidas. A DEA, IRS-CI e os departamentos de polícia locais desempenharam papéis cruciais. Os Farace agora enfrentam o confisco de todos os Bitcoins apreendidos.

### Olhando para o Futuro: Além da Cortina de Bitcoin
À medida que a saga dos Farace chega ao fim, nos deixa a refletir sobre a dança intricada entre crime e criptomoedas. Como uma dupla de pai e filho tentou ludibriar as autoridades, e o que está além dos Bitcoins confiscados? Acompanhe-nos para ficar por dentro de histórias intrigantes como essa.

---

Ei, Builder, deseja se manter por dentro das notícias mais relevantes da Web3? O fórum **360° News** 📢 foi criado para ajudar você a encontrar as notícias mais importantes da semana.  🗞️✨ 

Conecte-se ao nosso servidor no Discord, onde manteremos você informado sobre os principais acontecimentos. 

**[Clique aqui para acessar](https://discord.gg/web3dev)**

**Comment by s4mp41xao** [comment]
### Copy de Repost

Não perca a matéria completa da trama complexa que envolve venda de drogas na Silk Road, e lavagem de dinheiro em Bitcoins! A história envolvente de condenação do pai e filho Joseph Farace e Ryan Farace aguarda sua leitura. Siga o link abaixo para mergulhar na narrativa intrigante que revela um grande esquema criminoso no submundo da darknet.

**Comment by s4mp41xao** [comment]
### Fontes

https://www.justice.gov/usao-md/pr/father-and-son-sentenced-laundering-drug-trafficking-bitcoin-proceeds-intended-federal

https://www.irs.gov/compliance/criminal-investigation/father-and-son-sentenced-for-laundering-drug-trafficking-bitcoin-proceeds-intended-for-federal-forfeiture

https://cryptonews.net/news/legal/28382160/

https://www.theblock.co/post/274574/us-files-intent-to-dispose-117-million-in-bitcoin-seized-from-silk-road-drug-dealer

https://portaldobitcoin.uol.com.br/governo-dos-eua-ira-vender-us-117-milhoes-em-bitcoin-apreendidos-no-caso-silk-road/

https://www.justice.gov/opa/pr/first-nationwide-undercover-operation-targeting-darknet-vendors-results-arrests-more-35

**Comment by s4mp41xao** [comment]
# Bug Crítico da Nethermind interrompe o Processamento de Blocos na Rede Ethereum 

#hotfix #eth #nethermind

Essa semana, a empresa de infraestrutura Ethereum, Nethermind, confrontou-se com um bug crítico em várias versões de seu cliente de execução, prejudicando os usuários no processamento de blocos na rede Ethereum. 

O bug, introduzido na versão 1.23.0, foi prontamente solucionado com o lançamento da versão [1.25.2](https://github.com/NethermindEth/nethermind/releases/tag/1.25.2) Os usuários foram instados a atualizar para esta versão sem a necessidade de uma ressincronização.

Embora a Nethermind represente apenas 8,2% dos clientes de execução da Ethereum, preocupações foram levantadas dentro da comunidade sobre a super dependência do Geth, que representa 84%. 

O incidente destacou a importância de manter um ecossistema diversificado de clientes para a estabilidade e segurança da rede. [Leia o artigo completo](https://t.co/ZjEMJ0uG1o) reportado em nosso site.

**Comment by allluz** [comment]
@s4mp41xao o texto ficou ótimo mas fiquei um pouco em dúvida o enfoque dessa notícia. Dentre as fontes, acredito que a manchete sobre [a apreensão do governo](https://portaldobitcoin.uol.com.br/governo-dos-eua-ira-vender-us-117-milhoes-em-bitcoin-apreendidos-no-caso-silk-road/) poderia ser o foco como resolução do caso mencionado. Talvez alterando o foco e ordem dos parágrafos, a chamada poderia ser para a resolução do destino dos Bitcoins ao invés do caso ilícito @annabida como avalia?

**Comment by s4mp41xao** [comment]
O enredo principal mesmo é a visão geral dessa investigação, podemos minimizar a atenção das palavras nos títulos ou retirar para manter um formato mais analítico e descritivo do fato.

**Comment by s4mp41xao** [comment]
Eu mesmo depois da sua pontuação notei que dei muita ênfase em “Colaboração Pai e Filho: Lavagem atrás das Grades” ao comentar “Sem se deixar abalar pelas paredes da prisão”, concordo que podemos minimizar esse foco.

**Comment by s4mp41xao** [comment]
Concordo também em mudar o título principal como, por exemplo, “O Desfecho dos Bitcoins apreendidos em negociações proibidas no caso da Silk Road.”

**Comment by s4mp41xao** [comment]
Por mim eu só não mudaria a ordem dos parágrafos porque segue fielmente todo o contexto da divulgação oficial, mas eu concordaria sem problema com a decisão de vocês.

**Comment by s4mp41xao** [comment]
> [@s4mp41xao](https://github.com/s4mp41xao) perfeito. Trata-se de ordenar os fatos mais recentes sobre a decisão judicial e ajustar um pouco no título principal

Sim, exatamente. Buscarei títulos alternativos, deixei um exemplo mais abaixo.

**Comment by s4mp41xao** [comment]
# Ação das Autoridades: Desfecho Próximo para os Bitcoins Apreendidos em Negociações Proibidas na Silk Road

**Comment by s4mp41xao** [comment]
Publicado:

https://pt.w3d.community/blog/acao-das-autoridades-desfecho-proximo-para-os-bitcoins-apreendidos-em-negociacoes-proibidas-na-silk-road-lke
